components/bpmn/org.wso2.carbon.bpmn.extensions/src/main/java/org/wso2/carbon/bpmn/extensions/rest/RESTTask.java

```diff
@@ -352,6 +357,16 @@ private String getTaskDetails(DelegateExecution execution) {
         return task;
     }
 
+    private void setErrorDetailsForTaskExecution(DelegateExecution execution, String errorMessage,
+            RESTResponse response) {
+        if (httpStatusVariable != null && response != null) {
+            execution.setVariable(httpStatusVariable.getValue(execution).toString(), response.getHttpStatus());
+        }
+        if (errorMessageVariable != null) {
```

Comment:
Just in case if user haven't set errorMessageVariable in the BPMN, shall we populate this error message to default variable called ERROR_MESSAGE, ERROR_ACTIVITY_ID variables?

Reply:
fixed with 3da72be8c51273edd987f9d86e2749c250a9269d